scripts/zones/Lufaise_Meadows/mobs/Padfoot.lua

```diff
@@ -11,12 +11,19 @@ local ID = require("scripts/zones/Lufaise_Meadows/IDs")
 -----------------------------------
 local entity = {}
 
-entity.onMobSpawn = function(mob)
-    if mob:getID() == ID.mob.PADFOOT[GetServerVariable("realPadfoot")] then
-        mob:setDropID(2911)
-    else
-        mob:setDropID(1972)
-    end
+entity.onMobInitialize = function(mob)
+    mob:addListener("ITEM_DROPS", "ITEM_DROPS_PADFOOD", function(mobArg, loot)
+        if mob:getID() == ID.mob.PADFOOT[GetServerVariable("realPadfoot")] then
+            loot:addGroup(xi.loot.rate.GUARANTEED,
+            {
+                { item = xi.items.ASSAILANTS_RING, weight = 75 },
+                { item = xi.items.ASTRAL_EARRING, weight = 25 },
+            })
+        else
+            loot:addItem(xi.items.SHEEPSKIN, xi.loot.rate.GUARANTEED)
```

Comment:
I've actually recently killed this guy without getting the sheep sin on his fakes. just the cube.

Reply:
Teo confirmed! Reduced to very common.